custom_components/spotcast/__init__.py

```diff
@@ -245,6 +247,9 @@ def start_casting(call: ha_core.ServiceCall):
                 account, spotify_device_id, device_name, entity_id
             )
 
+        if (start_position):
```

Comment:
change to explicit bool check. truthy value is too broad and could lead to error. I assume in this case what we trully want to check is that `start_position` is not `None`?

Reply:
Done